docker/api_dockerfile

```diff
@@ -1,9 +1,13 @@
-FROM python:3.8-slim-buster
+FROM python:3.12-slim-bullseye
 
 WORKDIR /app
-RUN apt-get update && apt-get install -y build-essential python3-libgpiod
+RUN apt-get update && apt-get install -y build-essential
 COPY backend/requirements.txt .
+
 RUN pip3 install -r requirements.txt
 
+# python-libgpiod does not work on docker images
+# as it targets python 3.7 (??)
+RUN pip3 install gpiod
```

Comment:
Should we lock this to a specific version? I don't know what good practice for this sort of thing is in docker containers